src/main.jsx

```diff
@@ -2,29 +2,34 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 import { CommentSection } from './CommentSection'
 
-// Create a global function to initialize the comments
+// Create a function to initialize the comments
 console.log('Initializing Bluesky comments');
-window.initBlueskyComments = (elementId, uri) => {
+const initBlueskyComments = async (elementId, author) => {
```

Comment:
I think it would maybe make sense to support both versions? Either explicitly pass a post or use the author version? Though I'm open to being convinced it's unilaterally better to do it this way...

Reply:
Oh nvm just re-read hte PR description and saw you're gonna update this

Reply:
Definitely open to supporting both!